src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.username=username
```

Comment:
If the intention of the "dev" profile is to run the application locally, we can use "local" as profile name and name this file `application-local.properties`.

Though local development can be done using Testcontainers based setup, I think this file can be useful if someone wants to point to an existing database.

For logging SQL queries, I think the better choice is to follow one of the following articles:
* https://vladmihalcea.com/log-sql-spring-boot/
* https://thorben-janssen.com/spring-data-jpa-logging/